jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java

```diff
@@ -587,8 +619,42 @@ public void removeListener(Listener listener)
     public void doStart() throws Exception
     {
         if (LOG.isDebugEnabled())
-            LOG.debug("Scanner start: rprtExists={}, depth={}, rprtDirs={}, interval={}, filter={}, scannables={}",
-                      _reportExisting, _scanDepth, _reportDirs, _scanInterval, _filter, _scannables);
+            LOG.debug("Scanner start: autoStartScanning={}, reportExists={}, depth={}, rprtDirs={}, interval={}, filter={}, scannables={}",
+                isAutoStartScanning(), _reportExisting, _scanDepth, _reportDirs, _scanInterval, _filter, _scannables);
+
+        if (isAutoStartScanning())
+        {
+            // need to manually start _scheduler here
+            if (!_scheduler.isRunning())
+                _scheduler.start();
+            startScanning();
+        }
+
+        // Start the scanner and managed beans (eg: the scheduler)
+        super.doStart();
```

Comment:
We should not need to manually start the scheduler.  Let super do it:

```suggestion
        // Start the scanner and managed beans (eg: the scheduler)
        super.doStart();
        
        if (isAutoStartScanning())
            startScanning();
```